kubernetes/spec/support/e2e/api_discovery.rb

```
@@ -91,13 +131,32 @@ def discover_resources(group, version)
         payload = body.is_a?(String) ? JSON.parse(body) : body
-        resource_value(payload, "resources") || []
+        @resource_cache[cache_key] = resource_value(payload, "resources") || []
       rescue StandardError => e
+        raise if e.respond_to?(:code)
+
         raise "Failed to discover resources for #{group}/#{version}: #{e.message}"
       end
 
       private
 
+      def resource_served_strict?(group, version, kind)
```

RuboCop (1.87.0)

[convention] Use 2 (not 0) spaces for indented_internal_methods indentation.

(Layout/IndentationWidth)